python/cuml/test/dask/test_random_forest.py

```diff
@@ -13,24 +13,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+import pytest
 
 from cuml.dask.ensemble import RandomForestClassifier as cuRFC_mg
 from cuml.dask.ensemble import RandomForestRegressor as cuRFR_mg
+from cuml.dask.common import utils as dask_utils
 from sklearn.datasets import make_regression, make_classification
 from sklearn.model_selection import train_test_split
 from sklearn.metrics import accuracy_score, r2_score
 from dask.distributed import Client
 from dask_cuda import LocalCUDACluster
 
 import dask_cudf
+import dask_cuda
 import cudf
 import numpy as np
 import pandas as pd
 
 
-def test_rf_classification():
+@pytest.mark.parametrize('n_workers', [1,4,8])
```

Comment:
We should also test with 2 GPUs since that might be the most common number of GPUs > 1, and also what I have locally :P, so it'll increase our number of times that these tests run for n_gpus>1

```suggestion
@pytest.mark.parametrize('n_workers', [1, 2, 4, 8])
```